aspnet/identity/overview/migrations/migrating-an-existing-website-from-sql-membership-to-aspnet-identity.md

```diff
@@ -24,33 +24,33 @@ For this tutorial, we will take a web application template (Web Forms) created u
 
-    ![](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image2.png)
+    ![Image creating a new role](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image2.png)
 4. Create an Admin section of the site with a Default.aspx. Set the authorization tag in the web.config file to enable access only to users in Admin roles. More information can be found here [https://www.asp.net/web-forms/tutorials/security/roles/role-based-authorization-cs](../../../web-forms/overview/older-versions-security/roles/role-based-authorization-cs.md)
 
-    ![](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image3.png)
+    ![Image of newly created admin section](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image3.png)
 5. View the database in Server Explorer to understand the tables created by the SQL membership system. The user login data is stored in the aspnet\_Users and aspnet\_Membership tables, while role data is stored in the aspnet\_Roles table. Information about which users are in which roles is stored in the aspnet\_UsersInRoles table. For basic membership management it is sufficient to port the information in the above tables to the ASP.NET Identity system.
 
-    ![](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image4.png)
+    ![Image of the database in Server Explorer](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image4.png)
 
 ### Migrating to Visual Studio 2013
 
 1. Install Visual Studio Express 2013 for Web or Visual Studio 2013 along with the [latest updates](https://www.microsoft.com/download/details.aspx?id=45326).
 2. Open the above project in your installed version of Visual Studio. If SQL Server Express is not installed on the machine, a prompt is displayed when you open the project, since the connection string uses SQL Express. You can either choose to install SQL Express or as work around change the connection string to LocalDb. For this article we'll change it to LocalDb.
 3. Open web.config and change the connection string from .SQLExpress to (LocalDb)v11.0. Remove 'User Instance=true' from the connection string.
 
-    ![](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image3.jpg)
+    ![Image showing the connection string](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image3.jpg)
 4. Open Server Explorer and verify that the table schema and data can be observed.
 5. The ASP.NET Identity system works with version 4.5 or higher of the framework. Retarget the application to 4.5 or higher.
 
-    ![](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image5.png)
+    ![Image of A S P dot Net identity system and target framework](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image5.png)
```

Comment:
ASP

Reply:
I think it's important to be consistent with the article where screen readers are also used.

Reply:
change made